src/inet/tests/TestInetErrorStr.cpp

```diff
@@ -76,7 +76,7 @@ static void CheckInetErrorStr(nlTestSuite * inSuite, void * inContext)
     Inet::RegisterLayerErrorFormatter();
 
     // For each defined error...
-    for (int err : sContext)
+    for (int32_t err : sContext)
```

Comment:
Just as a warning, this will conflict with #8383...  Just something to watch out for.